src/UserSystem/Interfaces/IUserRepository.cs

```diff
@@ -0,0 +1,10 @@
+namespace Interfaces;
+
+public interface IUserRepository
+{
+    List<User> GetAllUsers();
```

Comment:
Also, I want to ask you about User class. Why do you consider to use it instead of Patron one?